src/components/SettingsPage/sections/CryptoAssets/Rates.js

```diff
@@ -82,25 +87,31 @@ class Rates extends PureComponent<Props> {
         <Body>
           <RateRowWrapper>
             <Box ff="Open Sans|SemiBold" alignItems="center" horizontal color="dark" fontSize={4}>
-              {'Rate'}
+              <Trans i18nKey="settings.rates.rate" />
               <TooltipButtonWrapper>
                 <Tooltip render={RateTooltip}>
                   <IconInfoCircle size={12} />
                 </Tooltip>
               </TooltipButtonWrapper>
             </Box>
             <Box ff="Open Sans|SemiBold" color="dark" fontSize={4}>
-              {'Price'}
+              <Trans i18nKey="settings.rates.rpice" />
             </Box>
             <Box ff="Open Sans|SemiBold" color="dark" fontSize={4}>
-              {'Last 30 days'}
+              <Trans i18nKey={`settings.rates.last`} values={{ range }} />
```

Comment:
mmh, i think we already have wording for the ranges. Can we use it?
also, the problem is you are not translating `range` here

Reply:
I'm not translating the range, because in that case `range` is the number of days. Probably not the best var name. The literal is `Last {{range}} days`